demo/parameter_files/pmp_input_parameters_demo1.py

```diff
@@ -1,8 +1,11 @@
 ################################################################################
-#  SAMPLE INPUT PARAMETER FILE FOR THE PCMDI METRICS PACKAGE (PMP V1.1)
+#  THIS IS A DEMO INPUT PARAMETER FILE FOR THE PCMDI METRICS PACKAGE (PMP V1.1)
 #  THIS IS A VERY SIMPLE EXAMPLE, ONLY COMPUTING STATISTICS FOR ONE MODEL VERSION AND ONE VARIABLE 
 #  SAMPLE MODEL AND OBSERVATIONAL DATA TO RUN THIS CODE ARE AVAILABLE FROM: 
-# 
+#  http://oceanonly.llnl.gov/gleckler1/pmp-demo-data/pmpv1.1_demodata.tar
```

Comment:
If you want to improve the demo that would be really great.  But 3 things to keep in mind: 1) think python newbie, 2) keep as simple as possible, and 3) real documentation  ;-)


From: Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>>
Reply-To: PCMDI/pcmdi_metrics <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, July 19, 2016 at 1:42 PM
To: PCMDI/pcmdi_metrics <pcmdi_metrics@noreply.github.com<mailto:pcmdi_metrics@noreply.github.com>>
Cc: gleckler1 <pgleckler@llnl.gov<mailto:pgleckler@llnl.gov>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [PCMDI/pcmdi_metrics] 343 pjg simple demos (#386)


In demo/parameter_files/pmp_input_parameters_demo1.py<https://github.com/PCMDI/pcmdi_metrics/pull/386#discussion_r71419133>:

>  #  THIS IS A VERY SIMPLE EXAMPLE, ONLY COMPUTING STATISTICS FOR ONE MODEL VERSION AND ONE VARIABLE
>  #  SAMPLE MODEL AND OBSERVATIONAL DATA TO RUN THIS CODE ARE AVAILABLE FROM:
> -#
> +#  http://oceanonly.llnl.gov/gleckler1/pmp-demo-data/pmpv1.1_demodata.tar


@gleckler1<https://github.com/gleckler1> I think the demo should do this for them, or at least try to. Do you want me to set this up?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/PCMDI/pcmdi_metrics/pull/386/files/628a8b02e66a9688e975e40f3b67531aa7d878f3#r71419133>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEV6rYOFYoXZ44nkL1Ie6M3axvk5Glbeks5qXTafgaJpZM4JQJb4>.